documentation/modules/rn-2.5.adoc

```diff
@@ -0,0 +1,199 @@
+
+.Unable to schedule migrated VM with multiple disks to more than one storage classes of type hostPath
+
+When migrating a VM with multiple disks to more than one storage classes of type hostPath, it may result in a VM that cannot be scheduled. Workaround: It is recommended to use shared storage on the target OCP cluster.
+
+.Non-supported guest operating systems in warm migrations
+
+Warm migrations and migrations to remote OCP clusters from vSphere do not support all types of guest operating systems that are supported in cold migrations to the local OCP cluster. It is a consequence of using RHEL 8 in the former case and RHEL 9 in the latter case. +
+See link:https://access.redhat.com/articles/1351473[Converting virtual machines from other hypervisors to KVM with virt-v2v in RHEL 7, RHEL 8, and RHEL 9] for the list of supported guest operating systems.
+
+.VMs from vSphere with RHEL 9 guest operating system may start with network interfaces that are down
+
+When migrating VMs that are installed with RHEL 9 as guest operating system from vSphere, their network interfaces could be disabled when they start in {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-491[(MTV-491)]
+
+.Import OVA: ConnectionTestFailed message appears when adding OVA provider
+
+When adding an OVA provider, the error message `ConnectionTestFailed` may instantly appear, although the provider is created successfully. If after a few minutes, the message does not disappear and the provider status does not move to `ready` condition, the `ova server pod creation` has failed. link:https://issues.redhat.com/browse/MTV-671[MTV-671]
+
+.OVA migration with multiple disks is not functioning
+
+OVA migration with multiple disks is not functioning as expected. When trying to migrate an OVA VM, with more than one disk, the migration becomes stuck at the `allocate disks` phase. link:https://issues.redhat.com/browse/MTV-676[MTV-676]
+
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/browse/MTV-562?filter=12419159[Known Issues] in Jira.
+
+[id="resolved-issues-25_{context}"]
+== Resolved issues
+
+// are there any resolved issues you want to highlight in this release?
+.Ensure up-to-date data is displayed in the create and update provider form
+
+In previous releases of {project-short}, the create and update provider forms could have presented stale data.
+
+This issue is resolved in {project-short} {project-version}, the new forms of create and update provider display up-to-date properties of the provider. link:https://issues.redhat.com/browse/MTV-603[MTV-603]
+
+.Snapshots that are created during the migration in OpenStack are not deleted
+
+In previous releases of {project-short}, the Migration Controller service did not delete snapshots that were created during the migration for source virtual machines in OpenStack automatically.
```

Comment:
```suggestion
In previous releases of {project-short}, the Migration Controller service did not delete snapshots that were created during the migration of source virtual machines in OpenStack automatically.
```
or maybe we should rather say ".. that were created automatically during the migration from OpenStack" ?